.clusterfuzzlite/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update && apt-get install -y make autoconf automake libtool
+
+COPY . $SRC/libuv
+COPY .clusterfuzzlite/build.sh $SRC/build.sh
+WORKDIR $SRC/libuv
```

Comment:
This file is unused, right?

Reply:
Do you mean the Dockerfile as a whole? or which which file are you referencing?

The Dockerfile is needed for [ClusterFuzzLite](https://google.github.io/clusterfuzzlite/) to work